testing/sdk_marathon.py

```diff
@@ -225,22 +231,29 @@ def _update():
 
 def destroy_app(app_name: str, timeout=TIMEOUT_SECONDS) -> None:
     @retrying.retry(stop_max_delay=timeout * 1000, wait_fixed=2000)
-    def _destroy():
+    def _destroy() -> dict:
         response = sdk_cmd.cluster_request(
             "DELETE", _api_url("apps/{}".format(app_name)), params={"force": "true"}
         )
-        return _handle_marathon_deployment_response(response)
+        return response.json()
 
     result = _destroy()
-    result.raise_on_error()
+    deployment_id = result["deploymentId"]
 
     # This check is different from the other deployment checks.
     # When it's complete, the app is gone entirely.
     @retrying.retry(
         stop_max_delay=timeout * 1000, wait_fixed=2000, retry_on_result=lambda result: not result
     )
     def _wait_for_app_destroyed():
-        return not app_exists(app_name, timeout)
+        running_deployments = sdk_cmd.cluster_request("GET", _api_url("deployments")).json()
```

Comment:
Should this check not include *both* a `not app_exists` call as well as a wait for empty deployments?

Reply:
I don't think `app_exists` is very helpful here because when Marathon receives the event, it immediately removes the `/apps/{name}` entry because the desired state is `removed`.

Reply:
Discussed in person and will be addressed.